### PR TITLE
Feat/ts tests

### PR DESCRIPTION
**Pull request by jtblack-aws**
**Issue #, if available:** n/a

### Testing

Before you submit a pull request, please make sure you have to following:

- [ ] Your code contains tests that cover all new code and changes
- [ ] All new and existing tests passed
- [ ] I have read the [Contributing Guidelines](https://github.com/aws-solutions-library-samples/guidance-for-overhead-imagery-inference-on-aws/blob/main/CONTRIBUTING.md) and agree to follow the [Code of Conduct](
https://github.com/aws-solutions-library-samples/guidance-for-overhead-imagery-inference-on-aws/blob/main/CODE_OF_CONDUCT.md))

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -47,7 +48,7 @@
     "eslint-plugin-simple-import-sort": "^8.0.0",
     "jest": "^29.7.0",
     "lint-staged": "^13.3.0",
-    "osml-cdk-constructs": "^1.6.0",
+    "osml-cdk-constructs": "file:lib/osml-cdk-constructs",
```
Comment (RanbirAulakh) [inline package.json:51]: You might want to undo this and update to the latest version.

File: bin/deploy-tile-server.ts [hunk 2]
```diff
@@ -59,4 +62,38 @@ export function deployTileServer(
...
+    `${targetAccount.name}-TSTestRunner`,
+    {
+      env: targetEnv,
+      account: targetAccount,
+      osmlVpc: vpcStack.resources,
+      tsEndpoint: tsEndpoint,
+      tsTestImageBucket: tsTestImageryBucket,
+      buildFromSource: buildFromSource,
+      description:
+        "Tile Server test runner, Guidance for Overhead Imagery Inference on AWS (SO9240)"
```
Comment (RanbirAulakh) [inline bin/deploy-tile-server.ts:94]: I am thinking, maybe in a separate PR. Anything that's inside a `model_runner` should be named as 

> Model Runner, Guidance for Overhead Imagery Inference on AWS (SO9240)

Then anything inside `tile_server` should be named as

> Tile Server, Guidance for Overhead Imagery Inference on AWS (SO9240)

It makes it clean and consistent.

File: lib/osml-stacks/tile_server/testing/ts-imagery.ts [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+  env: Environment;
+  account: OSMLAccount;
+  vpc: IVpc;
+}
+
+export class TSImageryStack extends Stack {
+  public resources: TSImagery;
+
+  /**
+   * Constructor for the model runner test imagery deployment cdk stack
```
Comment (RanbirAulakh) [inline lib/osml-stacks/tile_server/testing/ts-imagery.ts:19]: Update `model runner` to `tile server`

File: lib/osml-stacks/tile_server/testing/ts-imagery.ts [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+   * @param parent the parent cdk app object
+   * @param name the name of the stack to be created in the parent app object.
+   * @param props the properties required to create the stack.
+   * @returns the created MRImageryStack object
```
Comment (RanbirAulakh) [inline lib/osml-stacks/tile_server/testing/ts-imagery.ts:23]: Update `MRImageryStack` to `TSImageryStack`

File: package.json (hunk 1 above)
Comment (jtblack-aws) [reply package.json:51]: Yep! just part of testing before the CDK was published.  Good call-out!

File: lib/osml-stacks/tile_server/testing/ts-imagery.ts (hunk 3 above)
Comment (jtblack-aws) [reply lib/osml-stacks/tile_server/testing/ts-imagery.ts:19]: Done!

File: bin/deploy-tile-server.ts (hunk 2 above)
Comment (jtblack-aws) [reply bin/deploy-tile-server.ts:94]: I was going to propose this myself!  I'll update this file to match that convention and then we can have another PR to align the rest of them.

**Pull request closed without merging**